### PR TITLE
Packaging of application done for Linux

### PR DESCRIPTION
**Pull request by SaurusXI**
Switched to fman-build-system (FBS) for packaging. FBS needed restructuring of the project. Most of the source code is still the same, and can be found in ```src/main/python```
Assets have been moved to ```src/main/resources```
Finally, a markdown file with installation instructions has been added.

### CODE REVIEW

**Comment by SaurusXI** [comment]
The GitHub Actions workflows are still broken and need to be fixed. I'm working on that. Everything else is working though.

File: src/sign/linux/private-key.gpg [hunk 1]
```diff
@@ -0,0 +1,107 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
+
```
Comment (L04DB4L4NC3R) [inline src/sign/linux/private-key.gpg:2]: Remove this. GPG key should not be public

File: src/sign/linux/public-key.gpg [hunk 2]
```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
```
Comment (L04DB4L4NC3R) [inline src/sign/linux/public-key.gpg:2]: Remove this as well

File: .github/scripts/mac/publish.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+
+set timeout -1
+spawn fbs login
+expect "Username:"
+send -- "SaurusXI\r"
+expect "Password:"
+send -- "YeahScienceBitch\r"
```
Comment (L04DB4L4NC3R) [inline .github/scripts/mac/publish.sh:8]: Should not be public

File: .github/scripts/ubuntu/login.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/expect -f
+
+set timeout -1
+spawn fbs login
+expect "Username:"
+send -- "SaurusXI\r"
+expect "Password:"
+send -- "YeahScienceBitch\r"
```
Comment (L04DB4L4NC3R) [inline .github/scripts/ubuntu/login.sh:8]: Should not be public

File: .github/scripts/ubuntu/release.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/expect -f
+
+set timeout -1
+spawn fbs login
+expect "Username:"
+send -- "SaurusXI\r"
+expect "Password:"
+send -- "YeahScienceBitch\r"
```
Comment (L04DB4L4NC3R) [inline .github/scripts/ubuntu/release.sh:8]: Should not be public

**Comment by SaurusXI** [comment]
Done.

**Pull request merged**